_episodes/12-cmdline.md

```diff
@@ -873,13 +873,17 @@ the program now does everything we set out to do.
 > >
 > > def main():
 > >     script = sys.argv[0]
-> >     action = sys.argv[1]
-> >     if action not in ['--min', '--mean', '--max']: # if no action given
-> >         action = '--mean'    # set a default action, that being mean
-> >         filenames = sys.argv[1:] # start the filenames one place earlier in the argv list
+> >     if len(sys.argv) == 1: # there are no args but there may be a pipe
+> >         action = '--mean'  # set default action to --mean
+> >         filenames = []     # try stdin
 > >     else:
-> >         filenames = sys.argv[2:]
-> >
+> >         action = sys.argv[1] # it's safe now to get action argument
+> >         if action not in ['--min', '--mean', '--max']: # if other action given
```

Comment:
Here the comment was changed from "If no action given" to "if other action given" which seems a minor tweak, but in the case of another action (something made up or a typo) given vs. no action given, moving up the index to get filenames will error on an `OSError` when the other action is treated as a filename. I don't think we necessarily need to remedy this possible error too for the sake of the exercise, however, I think it would be good for you to change the comment back to "if no action given".